File: lista/Python/questao1.py
"""
1. Crie uma classe Carro com atributos como marca, modelo, e ano.
Instancie três objetos dessa classe e exiba os detalhes de cada um.
"""
class Carro:
    def __init__(self, marca, modelo, ano):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano

    def exibir_detalhes(self):
        print(f"Carro: {self.marca} {self.modelo}, Ano: {self.ano}")

def main():
    carro1 = Carro("Fiat", "Uno", 2020)
    carro2 = Carro("Chevrolet", "Onix", 2019)
    carro3 = Carro("Volkswagen,", "Gol", 2010)

    carro1.exibir_detalhes()
    carro2.exibir_detalhes()
    carro3.exibir_detalhes()

if __name__ == "__main__":
    main()

File: lista/GO/questao1/questao1.go
/*
Crie uma classe Carro com atributos como marca, modelo, e
ano. Instancie três objetos dessa classe e exiba os detalhes de cada um.
*/
package main

import (
	"fmt"
)

type Carro struct {
	Marca  string
	Modelo string
	Ano    int
}

func main() {
	carro1 := Carro{Marca: "Fiat", Modelo: "Uno", Ano: 2020}
	carro2 := Carro{Marca: "Chevrolet", Modelo: "Onix", Ano: 2020}
	carro3 := Carro{Marca: "Volkswagen", Modelo: "Gol", Ano: 2010}

	fmt.Println("Detalhes do Carro 1:")
	fmt.Printf("Marca: %s, Modelo: %s, Ano: %d\n", carro1.Marca, carro1.Modelo, carro1.Ano)

	fmt.Println("Detalhes do Carro 2:")
	fmt.Printf("Marca: %s, Modelo: %s, Ano: %d\n", carro2.Marca, carro2.Modelo, carro2.Ano)

	fmt.Println("Detalhes do Carro 3:")
	fmt.Printf("Marca: %s, Modelo: %s, Ano: %d\n", carro3.Marca, carro3.Modelo, carro3.Ano)
}
